v2/controllers/marketplace/meterbase_controller.go

```diff
@@ -646,9 +812,20 @@ func (r *MeterBaseReconciler) reconcilePrometheusOperator(
 		Patcher: r.patcher,
 	}
 
-	watchNamespace, _ := getWatchNamespace()
```

Comment:
Why do we not use the getWatchNamespace function? Think this may break our operator deployment.

Reply:
watchNamespace was used to feed the factory for NewPrometheusOperatorDeployment
NewPrometheusOperatorDeployment was changed
the nsValues are now the deny-namespaces list, that is, the platform openshift- namespaces (openshift.io/cluster-monitoring0